### PR TITLE
Add missing rules to rules-sources.mdx

### PR DESCRIPTION
**Pull request by SrBrahma**
## Summary

Rules present in the https://github.com/biomejs/biome/discussions/3 but not in the docs

### CODE REVIEW

File: website/src/content/docs/linter/rules-sources.mdx [hunk 1]
```diff
@@ -94,6 +95,7 @@ Some **Biome** rules might **not** have options, compared to the original rule.
 | [no-unused-labels](https://eslint.org/docs/latest/rules/no-unused-labels) |[noUnusedLabels](/linter/rules/no-unused-labels) |
 | [no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars) |[noUnusedVariables](/linter/rules/no-unused-variables) |
 | [no-useless-catch](https://eslint.org/docs/latest/rules/no-useless-catch) |[noUselessCatch](/linter/rules/no-useless-catch) |
+| [no-useless-computed-key](https://eslint.org/docs/latest/rules/no-useless-computed-key) |[useLiteralKeys](/linter/rules/use-literal-keys) |
```
Comment (SrBrahma) [inline website/src/content/docs/linter/rules-sources.mdx:98]: image

File: website/src/content/docs/linter/rules-sources.mdx [hunk 2]
```diff
@@ -52,6 +52,7 @@ Some **Biome** rules might **not** have options, compared to the original rule.
 | [no-constructor-return](https://eslint.org/docs/latest/rules/no-constructor-return) |[noConstructorReturn](/linter/rules/no-constructor-return) |
 | [no-control-regex](https://eslint.org/docs/latest/rules/no-control-regex) |[noControlCharactersInRegex](/linter/rules/no-control-characters-in-regex) |
 | [no-debugger](https://eslint.org/docs/latest/rules/no-debugger) |[noDebugger](/linter/rules/no-debugger) |
+| [no-delete-var](https://eslint.org/docs/latest/rules/no-delete-var) |[noDelete](/linter/rules/no-delete) |
```
Comment (SrBrahma) [inline website/src/content/docs/linter/rules-sources.mdx:55]: image

**Comment by Conaclos** [comment]
Thanks for your fix. Unfortunately, this page is generated from our rules' metadata. Our rules' metadata cannot currently refer to several sources. This is why these references are missing.

File: website/src/content/docs/linter/rules-sources.mdx [hunk 3]
```diff
@@ -208,5 +211,7 @@ Some **Biome** rules might **not** have options, compared to the original rule.
 | ---- | ---- |
 | [no-array-for-each](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-for-each.md) |[noForEach](/linter/rules/no-for-each) (inspired) |
 | [no-instanceof-array](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-instanceof-array.md) |[useIsArray](/linter/rules/use-is-array) |
+| [no-static-only-class](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-static-only-class) |[noStaticOnlyClass](/linter/rules/no-static-only-class) |
+| [no-typeof-undefined](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-typeof-undefined) |[useValidTypeof](/linter/rules/use-valid-typeof) |
```
Comment (SrBrahma) [inline website/src/content/docs/linter/rules-sources.mdx:215]: image

File: website/src/content/docs/linter/rules-sources.mdx [hunk 4]
```diff
@@ -78,6 +79,7 @@ Some **Biome** rules might **not** have options, compared to the original rule.
 | [no-param-reassign](https://eslint.org/docs/latest/rules/no-param-reassign) |[noParameterAssign](/linter/rules/no-parameter-assign) |
 | [no-prototype-builtins](https://eslint.org/docs/latest/rules/no-prototype-builtins) |[noPrototypeBuiltins](/linter/rules/no-prototype-builtins) |
 | [no-regex-spaces](https://eslint.org/docs/latest/rules/no-regex-spaces) |[noMultipleSpacesInRegularExpressionLiterals](/linter/rules/no-multiple-spaces-in-regular-expression-literals) |
+| [no-return-assign](https://eslint.org/docs/latest/rules/no-return-assign) |[noAssignInExpressions](/linter/rules/no-assign-in-expressions) |
```
Comment (SrBrahma) [inline website/src/content/docs/linter/rules-sources.mdx:82]: image

**Comment by SrBrahma** [comment]
> Thanks for your fix. Unfortunately, this page is generated from our rules' metadata. Our rules' metadata cannot currently refer to several sources. This is why these references are missing.

@Conaclos I understand. Maybe I can try to make a PR to allow multiple sources? If you think it's a good and possible idea, I would appreciate to know which are the main files to make that change.

**Comment by ematipico** [comment]
Why do you think we should show many sources for one rule? 

The initial idea was to give an understanding of how the rule works in an Eslint environment, and if Biome rules. 

If we put multiple rules, we also need to spend time to understand if each source does the same as the other sources. 

If we need to have a very strong reason to justify such Maintenance burden

**Comment by SrBrahma** [comment]
> Why do you think we should show many sources for one rule?

Yesterday I migrated the [eslint-config-biome](https://github.com/SrBrahma/eslint-config-biome) from the https://github.com/biomejs/biome/discussions/3 to the quite new docs page (https://github.com/SrBrahma/eslint-config-biome/commit/84d2108ac1fdd4a11371cc9aa85c10f0bb185354). The diff on the output is what made me make the changes present in this PR, to keep in line with what #3 gives as equivalencies, as in these situations that I made the changes, they two differ in their content.

I suggested to support multiple sources due to "Our rules' metadata cannot currently refer to several sources. This is why these references are missing.", as Conaclos said.

Not a big deal, so I will close this.

**Comment by Conaclos** [comment]
I will take a look when I will wirk on #1752

**Pull request closed without merging**